src/modules/role/role.service.ts

```diff
@@ -360,7 +364,9 @@ export class RoleService {
       );
       return;
     }
-    this.logger.debug(`### Updating DTO for namespace ${namespace}`);
+    this.logger.debug(
+      `### Updating DTO for namespace ${namespace}, DTO: ${JSON.stringify(dto)}`
+    );
```

Comment:
In my opinion, the `###` are noise in the logs
```suggestion
    this.logger.debug(
      `Updating DTO for namespace ${namespace}, DTO: ${JSON.stringify(dto)}`
    );
```